### PR TITLE
chore: Migrates ControlHeader icons

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Migrates `ControlHeader` icons.

@pkdotson 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Screen Shot 2021-06-18 at 2 30 07 PM
Screen Shot 2021-06-18 at 2 19 46 PM
Screen Shot 2021-06-18 at 2 30 43 PM
Screen Shot 2021-06-18 at 2 27 38 PM

### TESTING INSTRUCTIONS
1 - Go to explore
2 - Change the controls to show the icons

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by pkdotson** [comment]
@michael-s-molina  did the new icons get design signoff? I notice they are a little different but that's probably because the new icons aren't exactly the same.

**Review by pkdotson — APPROVED** [review]
LGTM!

**Comment by michael-s-molina** [comment]
> I notice they are a little different but that's probably because the new icons aren't exactly the same.

@pkdotson Exactly. The new warning and error icons are different.

**Comment by rusackas** [comment]
One small detail to consider... I noticed that the icons are a little high up, vertically. It seems the `anticon` class has a `vertical-align` of `-.125em` for some reason. If I override that with `vertical-align: middle;` it aligns much better with the text.  

vertalign2

While this _sounds_ like a sensible global change, I'm not sure what fallout/misalignments it might cause. We can pus this PR through as is, or stop and poke around at this. Your call. If you want to merge as is, maybe we can make a project board of "Things to try when we have visual regression testing"

**Comment by michael-s-molina** [comment]
> One small detail to consider... I noticed that the icons are a little high up, vertically. It seems the `anticon` class has a `vertical-align` of `-.125em` for some reason. If I override that with `vertical-align: middle;` it aligns much better with the text.
> 
> vertalign2
> 
> While this _sounds_ like a sensible global change, I'm not sure what fallout/misalignments it might cause. We can pus this PR through as is, or stop and poke around at this. Your call. If you want to merge as is, maybe we can make a project board of "Things to try when we have visual regression testing"

Someone probably fixed this while changing another requirement. This is the current state with latest `master`:

Screen Shot 2021-06-25 at 3 10 04 PM

**Pull request merged**
